.gitpod.yml

```
@@ -1,21 +1,19 @@
-# Using an updated Gitpod docker image (when using a non default JDK or tools)
-image:
-  file: .gitpod.Dockerfile
-
 tasks:
-  - init: sdk default java  11.0.15-tem && mvn clean verify
+  - name: Preview
+    command: gp ports await 8080 && gp url 8080 && gp preview $(gp url 8080)/jenkins/
+
+  - name: Setup and Build
+    before: |
+      # Download and Install Java 17.0.6-tem
+      yes Y | sdk install java 17.0.6-tem
```

Is there a reason why this no longer sets the default SDK?

Also, is there a reason why only one SDK is installed instead of installing both Java 11 and Java 17?  No problem with the decision, just curious for the reasoning.
```suggestion
      yes Y | sdk install java 17.0.6-tem && sdk default java 17.0.6-tem
```

---

`sdk default java 17.0.6-tem` is not required as when we run the command `sdk install java 17.0.6-tem`, we get a prompt which asks if we want to set it as default or not.  For some reason ``sdk default java 17.0.6-tem`` does not work unless I manually answer the prompt.

> Also, is there a reason why only one SDK is installed instead of installing both Java 11 and Java 17? No problem with the decision, just curious for the reasoning.
> 
I don't have an answer for it as I thought only one SDK is required and Java 17 seems fine. But now I am curious to know why do you think I should have considered installing two SDKs

---

Thanks for the clarification on the behavior of the `sdk` command to set the first installed jdk as the default.  I wasn't aware of that.  Very nice.

I believe two JDK's were included initially because Jenkins development and execution are supported with either Java 11 or Java 17.  I don't see a reason to mandate that two JDK's must be installed, since the user of the Gitpod configuration could install JDK 11 after startup if they needed it.